test/MUXControls.Test/Infra/Application.cs

```diff
@@ -384,8 +392,9 @@ private bool CloseAppWindowWithCloseButton()
             UIObject closeAppInvoker;
             if (!topWindowObj.Descendants.TryFind(UICondition.Create("@AutomationId='__CloseAppInvoker'"), out closeAppInvoker))
             {
-                Log.Comment("Application.CloseAppWindowWithCloseButton: Failed to find close app invoker");
-                return false;
+                Log.Comment("Application.CloseAppWindowWithCloseButton: Failed to find close app invoker. Terminating app through process id.");
+                EnsureApplicationProcessHasExited(CoreWindow.ProcessId);
```

Comment:
I don't think this is necessary. We call CloseAppWindowWithCloseButton from Close, which already has logic to fallback to terminating the process.

Reply:
So just remove this change?

Reply:
Yeah. I filed #2311 a while ago, before I made these recent changes to Application.cs. So some of the things that 2311 says are no longer valid.

Reply:
I see, removed it now.